invenio_communities/token.py

```diff
@@ -16,84 +16,59 @@
 from flask import current_app
 from itsdangerous import BadData, JSONWebSignatureSerializer, SignatureExpired
 
-SUPPORTED_DIGEST_ALGORITHMS = ('HS256', 'HS512')
 
+def create_token(obj_id, extra_data={}):
```

Comment:
```suggestion
def create_token(payload=None):
```

and `payload = payload or {}` in the body

(see comment below for why `obj_id` is removed)